pkg/tfbridge/info/info.go

```diff
@@ -129,6 +129,9 @@ type Provider struct {
 	// - Name
 	// - TFProviderModuleVersion
 	//
+	// For dynamically bridged providers, when the fullDocs option is set on the parameterized inputs,
+	// this field will be automatically set to a temporary directory containing a shallow clone of the upstream repo.
```

Comment:
Who is this comment for?

Reply:
Maintainers. It's set implicitly here, so it makes sense to document this behavior.

Reply:
I wouldn't say its *set implicitly*. It is *set* in `dynamic/info.go`. If you are working in `/dynamic`, then you will see where it is set. If you are implementing a bridged provider, then this is not helpful for you, since it's relevant only to dynamic providers. `info/info` feels like the wrong place for this comment.

This feels spiritually equivalent to annotating a function with where it is called, which we don't and shouldn't do.